webui/react/src/hooks/useModal/Checkpoint/useModalCheckpoint.module.scss

```diff
@@ -6,6 +6,17 @@
 
   .row {
     display: flex;
+
+    .size {
+      align-items: center;
+      display: flex;
+
+      :global {
```

Comment:
I think that this :global section is not needed?

Reply:
oh, I thought it was the footer buttons, now I see that it is for the "Request Checkpoint Deletion" button, my bad